### PR TITLE
fix(core): Fix two panics

### PR DESCRIPTION
**Pull request by anurag-rajawat**
## Description

This PR fix panics due to interface conversion and ai handler in the sentryflow, resolving crash.

### CODE REVIEW

File: sentryflow-clients/.gitignore [hunk 1]
```diff
@@ -0,0 +1,27 @@
+### macOS
+.DS_Store
```
Comment (VedRatan) [inline sentryflow-clients/.gitignore:2]: will the .gitignore file in the root directory won't work?

**Review by VedRatan — CHANGES_REQUESTED** [review]

File: sentryflow-clients/.gitignore (hunk 1 above)
Comment (anurag-rajawat) [reply sentryflow-clients/.gitignore:2]: You're right, thanks for pointing out this.

**Review by VedRatan — APPROVED** [review]
LGTM

**Comment by VedRatan** [comment]
@anurag-rajawat it would be good if you could resolve the conflicts.

**Review by VedRatan — APPROVED** [review]
LGTM

**Pull request merged**